### PR TITLE
Adding Keep

### PR DESCRIPTION
**Pull request by talboren**
### What does this PR do?

Adding Keep as a marketplace tile in Datadog's marketplace.

### Motivation

DPN

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

-

### CODE REVIEW

File: keep/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
```
Comment (emarsha94) [inline keep/README.md:6]: The overview should include: 

1. What does your product do (in 1-2 sentences)?
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data?

File: keep/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
```
Comment (emarsha94) [inline keep/README.md:11]: Please include the steps for the oauth flow customers will take

File: keep/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+## Setup
+
+### Installation
```
Comment (emarsha94) [inline keep/README.md:9]: As mentioned in this [article](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client), please include uninstallation steps.

File: keep/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Keep does not include any service checks.
+
+### Events
+
+Keep does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][3].
```
Comment (emarsha94) [inline keep/README.md:37]: This should be the partners support team

File: keep/images/IMAGES_README.md
Comment (emarsha94) [inline keep/images/IMAGES_README.md:1]: This file should be deleted

File: keep/manifest.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        "media_type": "image",
+        "caption": "Keep's workflow builder page",
+        "image_url": "images/workflow-builder.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
```
Comment (emarsha94) [inline keep/manifest.json:44]: This should include all relevant fields as seen below as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags): 
      "Category::",
      "Offering::",
      "Queried Data Type::" If pulling data 
      "Submitted Data Type::",

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Thank you for submitting the PR! In addition to the comments, please include a dashboard, and a logo in svg format. For dashboards, please refer to our dashboard best practices: https://datadoghq.dev/integrations-core/guidelines/dashboards/

**Comment by talboren** [comment]
> Thank you for submitting the PR! In addition to the comments, please include a dashboard, and a logo in svg format. For dashboards, please refer to our dashboard best practices: https://datadoghq.dev/integrations-core/guidelines/dashboards/

Thank you for the review, @emarsha94! I've resolved all the comments (hopefully 🤞🏼) and added `keep-logo.svg` under images.
As for the dashboard, we still don't have an example for that and I couldn't find this as a requirement anywhere -- can we handle this later?

File: keep/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
+
+Users will:
+1. Be redirected to Keep's platform
+2. Click on the Datadog tile to initate an OAuth flow
+3. Review and confirm the required scopes in their Datadog account
```
Comment (emarsha94) [inline keep/README.md:20]: This should include the entire oauth flow a customer will take starting from the Datadog tile. Please include all steps a customer will take in the oauth flow

File: keep/README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+Keep does not include any service checks.
+
+### Events
+
+Keep does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog Partners Support][3] team.
```
Comment (emarsha94) [inline keep/README.md:52]: This should be specific to the partner. Ex: Need help? Contact Keep Support

File: keep/README.md (hunk 6 above)
Comment (talboren) [inline keep/README.md:20]: @emarsha94 I hope this makes sense, I couldn't find a great example somewhere else so I'm pretty much in the dark here 😆 

```suggestion
1. Clicking on Keep's integration tile -> Install Integration
2. User will be redirected to Keep's platform for sign-in
3. User will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
4. User will be redirected back to Keep's platform to see whether installation was successfully completed
```

File: keep/README.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+[1]: https://www.keephq.dev/
+[2]: https://app.datadoghq.com/account/settings/agent/latest
+[3]: https://docs.datadoghq.com/help/
```
Comment (talboren) [inline keep/README.md:56]:
```suggestion
Need help? Contact (Keep's support)[mailto:rnd@keephq.dev?subject=[Datadog]%20OAuth%20Integration%20Support] team.

[1]: https://www.keephq.dev/
[2]: https://app.datadoghq.com/account/settings/agent/latest
```

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -93,6 +93,7 @@
 /rum_javascript/                         @DataDog/rum-app
 /k6/                                     @ppcano support@k6.io
 /kernelcare/                             @grubberr schvaliuk@cloudlinux.com
+/keep/                                   @talboren tal@keephq.dev @DataDog/documentation
```
Comment (alai97) [inline .github/CODEOWNERS:96]: `documentation` is defined for reviews in Line 4 above  🙌🏼

```suggestion
/keep/                                   @talboren tal@keephq.dev @DataDog/ecosystems-review
```

File: keep/README.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+Keep acts as a single pane of glass and helps you turn 1000s into just 10s of meaningful alerts.
+
+We use monitors data, logs and events to correlate alerts and reduce noise.
```
Comment (alai97) [inline keep/README.md:9]:
```suggestion
Keep acts as a single pane of glass and helps you turn 1,000 seconds of alerts into 10 seconds of meaningful alerts. Keep uses Datadog monitor data, logs, and events to correlate alerts and reduce noise.
```

File: keep/README.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
```
Comment (alai97) [inline keep/README.md:15]:
```suggestion
To install the Keep integration, navigate to the [integration tile][1] on your Datadog account, and install the integration [using OAuth2][2].
```

File: keep/README.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+Users will:
+1. Clicking on Keep's integration tile -> Install Integration
+2. User will be redirected to Keep's platform for sign-in
+3. User will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
+4. User will be redirected back to Keep's platform to see whether installation was successfully completed
```
Comment (alai97) [inline keep/README.md:21]:
```suggestion

1. Click **Install Integration**.
2. Users are redirected to Keep's platform to sign in.
3. User are redirected back to Datadog to review and confirm the required scopes from their Datadog account.
4. User are redirected back to Keep's platform to see whether installation was successfully completed.
```

File: keep/README.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+## Uninstallation
+
+- Once this integration has been uninstalled, any previous authorizations are revoked. 
+- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the API Keys page.
+- Confirm that `Webhooks` integration does not contain `keep-datadog-webhook-integration-UUID` in the Datadog's (Integrations Page)[https://app.datadoghq.com/integrations].
```
Comment (alai97) [inline keep/README.md:27]:
```suggestion
- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][3].
- Confirm that `Webhooks` integration does not contain `keep-datadog-webhook-integration-UUID` on the [Integrations page][4].
```

File: keep/README.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+### Validation
+
+To validate Keep's integration is properly working, follow the next steps:
+1. Head over to Datadog's (Integrations Page)[https://app.datadoghq.com/integrations]
```
Comment (alai97) [inline keep/README.md:32]:
```suggestion
1. Navigate to the [Integrations page][4].
```

File: keep/README.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+2. Search for `Webhooks`
+3. Click the `Webhooks` tile
+4. In the installed `Webhooks` list, look for a `Webhook` that starts with `keep-datadog-webhook-integration-UUID`
```
Comment (alai97) [inline keep/README.md:35]:
```suggestion
2. Search for `Webhooks`.
3. Click the `Webhooks` tile.
4. In the installed `Webhooks` list, look for a `Webhook` that starts with `keep-datadog-webhook-integration-UUID`.
```

File: keep/README.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+## Data Collected
+
+### Metrics
+
+Keep sends metrics about the fatigueness of specific connected monitors.
```
Comment (alai97) [inline keep/README.md:41]:
```suggestion
Keep sends metrics about the fatigueness of specific connected monitors to Datadog. 
```

File: keep/README.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+## Data Collected
+
+### Metrics
+
+Keep sends metrics about the fatigueness of specific connected monitors. 
+
+### Service Checks
+
+Keep does not include any service checks.
```
Comment (alai97) [inline keep/README.md:45]:
```suggestion
The Keep integration does not include any service checks.
```

File: keep/README.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+
+### Events
+
+Keep does not include any events.
```
Comment (alai97) [inline keep/README.md:49]:
```suggestion
The Keep integration does not include any events.
```

File: keep/README.md [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+
+## Troubleshooting
+
+Need help? Contact (Keep's support)[mailto:rnd@keephq.dev?subject=[Datadog]%20OAuth%20Integration%20Support] team.
```
Comment (alai97) [inline keep/README.md:53]:
```suggestion
Need help? Contact [Keep's Support team][5].
```

File: keep/README.md [hunk 20]
```diff
@@ -0,0 +1,57 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
```
Comment (alai97) [inline keep/README.md:5]:
```suggestion
[Keep][6] is an open-source AIOps alert management tool.
```

File: keep/README.md [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+
+Keep does not include any events.
+
+## Troubleshooting
+
+Need help? Contact (Keep's support)[mailto:rnd@keephq.dev?subject=[Datadog]%20OAuth%20Integration%20Support] team.
+
+[1]: https://www.keephq.dev/
+[2]: https://app.datadoghq.com/account/settings/agent/latest
+
```
Comment (alai97) [inline keep/README.md:57]:
```suggestion
[1]: https://app.datadoghq.com/integrations/keephq
[2]: /developers/authorization/oauth2_in_datadog/
[3]: https://app.datadoghq.com/organization-settings/api-keys
[4]: https://app.datadoghq.com/integrations
[5]: mailto:rnd@keephq.dev?subject=[Datadog]%20OAuth%20Integration%20Support
[6]: https://www.keephq.dev/
```

File: keep/manifest.json [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "40ac95c0-35bd-49c8-a5f0-b21037bc87b4",
+  "app_id": "keep",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "An open-source AIOps Alert Management platform",
```
Comment (alai97) [inline keep/manifest.json:11]: Feel free to reword~

```suggestion
    "description": "Send Keep monitor metrics into Datadog.",
```

File: keep/manifest.json [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+    "title": "Keep",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Keep's providers page",
+        "image_url": "images/integrations-page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Keep's alerts single pane of glass",
```
Comment (alai97) [inline keep/manifest.json:21]:
```suggestion
        "caption": "Keep's alerts page",
```

**Review by alai97 — COMMENTED** [review]
Some copy + link suggestions!

File: keep/README.md (hunk 10 above)
Comment (anawishnoff) [inline keep/README.md:9]: Does this integration send data into Datadog at all? If so, can you explain the types of data/value of that data as well?

File: keep/README.md (hunk 13 above)
Comment (anawishnoff) [inline keep/README.md:27]: I don't see the Webhooks piece anywhere in the **Installation** instructions. Does this integration automatically create a webhook for the user?

Comment (anawishnoff) [reply keep/README.md:27]: Can we link directly to the [webhooks tile](https://app.datadoghq.com/integrations/webhooks) here?

File: keep/manifest.json (hunk 22 above)
Comment (anawishnoff) [reply keep/manifest.json:11]: Updated with my suggestion, but feel free to reword as Austin metnioned!

File: keep/manifest.json [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "An open-source AIOps Alert Management platform",
+    "title": "Keep",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Keep's providers page",
+        "image_url": "images/integrations-page.png"
+      },
```
Comment (anawishnoff) [inline keep/manifest.json:18]: I'd recommend removing this image, as we'd like for this integration tile to focus on the connection with Datadog

File: keep/manifest.json [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+      "Queried Data Type::Metrics",
+      "Queried Data Type::Logs",
+      "Queried Data Type::Events",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {},
+  "author": {
+    "support_email": "tal@keephq.dev",
+    "name": "Tal Borenstein",
```
Comment (anawishnoff) [inline keep/manifest.json:53]:
```suggestion
    "name": "Keep",
```

**Review by anawishnoff — DISMISSED** [review]
A few initial comments from me. 

Since this integration is sending in metrics to Datadog, a dashboard is required with this integration. You can see our [dashboard best practices guide here](https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard).

File: keep/manifest.json [hunk 26]
```diff
@@ -0,0 +1,58 @@
...
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "An open-source AIOps Alert Management platform",
+    "title": "Keep",
+    "media": [
+      {
```
Comment (anawishnoff) [inline keep/manifest.json:14]: You can add your video to the integration tile using the below syntax:

```suggestion
      {
        "media_type": "video",
        "caption": "<insert caption>",
        "image_url": "<path-to-thumbnail.jpg>",
        "vimeo_id": 906118097
      },
      {
```

File: keep/README.md (hunk 10 above)
Comment (talboren) [reply keep/README.md:9]: Hey @anawishnoff :) We plan on making it send our monitors "fatiguness meter" metric very soon. I'll update the documentation and the integration as soon as we do.

File: keep/README.md (hunk 13 above)
Comment (talboren) [reply keep/README.md]: You are right. I'll add that right now.

File: keep/README.md [hunk 27]
```diff
@@ -0,0 +1,72 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management and automation platform, which offers a platform for consolidating, automating, and reducing noise across various parts of your observability stack. Keep integration with Datadog harnesses the strengths of both platforms, providing a unified and efficient approach to alert management and event correlation.
```
Comment (emarsha94) [inline keep/README.md:5]:
```suggestion
Keep is an open-source AIOps alert management and automation platform, which offers a platform for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
```

File: keep/README.md [hunk 28]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
+
+Users will:
```
Comment (emarsha94) [inline keep/README.md:27]:
```suggestion
To setup the Keep integration
```

File: keep/README.md [hunk 29]
```diff
@@ -0,0 +1,72 @@
...
+1. Clicking on Keep's integration tile -> Install Integration
+2. User will be redirected to Keep's platform for sign-in
+3. User will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
+4. User will be redirected back to Keep's platform to see whether installation was successfully completed
```
Comment (emarsha94) [inline keep/README.md:31]:
```suggestion
1. Click on Keep's integration tile and select Install Integration
2. Users will be redirected to Keep's platform to sign-in to their Keep account
3. Users will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
4. Once confirmed, users will be redirected back to Keep's platform to see whether installation was successfully completed
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Thanks for updating the overview! Looks great.

File: keep/README.md [hunk 30]
```diff
@@ -0,0 +1,72 @@
...
+
+Keep is an open-source AIOps alert management and automation platform, which offers a platform for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
+
+Keep uses Datadog monitors data, logs and events to correlate alerts and reduce noise.
+
+Key features of this integration include:
+
+- Centralized Alert Management: Consolidate all your Datadog alerts into Keep’s single pane of glass, offering streamlined control and visibility.
+- Reduced Alert Noise: Minimize alert fatigue by filtering and prioritizing Datadog alerts, ensuring your team addresses the most critical alerts promptly.
+- Comprehensive Analysis: Leverage Keep’s analytical tools to derive insights from Datadog alerts, aiding in proactive decision-making and trend analysis.
```
Comment (alai97) [inline keep/README.md:13]:
```suggestion
- Centralized Alert Management: Consolidate all your Datadog alerts into Keep's single pane of glass, offering streamlined control and visibility.
- Reduced Alert Noise: Minimize alert fatigue by filtering and prioritizing Datadog alerts, ensuring your team addresses the most critical alerts promptly.
- Comprehensive Analysis: Leverage Keep's analytical tools to derive insights from Datadog alerts, aiding in proactive decision-making and trend analysis.
```

File: keep/README.md [hunk 31]
```diff
@@ -0,0 +1,72 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management and automation platform, which offers a platform for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
```
Comment (alai97) [inline keep/README.md:5]:
```suggestion
[Keep][6] is an open source AIOps alert management and automation platform, which offers capabilities for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
```

File: keep/README.md [hunk 32]
```diff
@@ -0,0 +1,72 @@
...
+
+Keep uses Datadog monitors data, logs and events to correlate alerts and reduce noise.
```
Comment (alai97) [inline keep/README.md:7]:
```suggestion
Keep uses Datadog monitors data, logs, and events to correlate alerts and reduce noise.
```

File: keep/README.md [hunk 33]
```diff
@@ -0,0 +1,72 @@
...
+
+Key features of this integration include:
+
+- Centralized Alert Management: Consolidate all your Datadog alerts into Keep’s single pane of glass, offering streamlined control and visibility.
+- Reduced Alert Noise: Minimize alert fatigue by filtering and prioritizing Datadog alerts, ensuring your team addresses the most critical alerts promptly.
+- Comprehensive Analysis: Leverage Keep’s analytical tools to derive insights from Datadog alerts, aiding in proactive decision-making and trend analysis.
+
+This integration is ideal for teams seeking to enhance their alert capabilities, improve operational efficiency, and make data-driven decisions with **reduced noise and distraction**.
+
+For more information about (Keep)[6], see (Keep's documentation)[8].
```
Comment (alai97) [inline keep/README.md:17]:
```suggestion
For more information, see the [official Keep documentation][8].
```

File: keep/README.md [hunk 34]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+## Setup
+
+### Installation
+
+The Keep integration is installed using the (Integration Tile)[https://app.datadoghq.com/integrations/keephq] on your Datadog account, with OAuth2.
```
Comment (alai97) [inline keep/README.md:25]:
```suggestion
Use the [integration tile ][9] on your Datadog account to install the Keep integration with [OAuth2][2].
```

File: keep/README.md [hunk 35]
```diff
@@ -0,0 +1,72 @@
...
+
+To setup the Keep integration
+1. Click on Keep's integration tile and select Install Integration
+2. Users will be redirected to Keep's platform to sign-in to their Keep account
+3. Users will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
+4. Once confirmed, users will be redirected back to Keep's platform to see whether installation was successfully completed
```
Comment (alai97) [inline keep/README.md:31]:
```suggestion
To set up the Keep integration:

1. Click on the Keep integration tile and select **Install Integration**.
2. Users are redirected to Keep's platform to sign into their Keep account.
3. Users are redirected back to Datadog to review and confirm the required scopes from their Datadog account.
4. Once confirmed, users are redirected back to the Keep platform to see whether installation was successfully completed.
```

File: keep/README.md [hunk 36]
```diff
@@ -0,0 +1,72 @@
...
+
+Once installed successfully, Keep will automatically create a new `Webhook` integration in your Datadog, and will modify your monitors to send alerts.
```
Comment (alai97) [inline keep/README.md:33]:
```suggestion
Once installed successfully, Keep automatically creates a new `Webhook` integration in Datadog, and modifies your monitors to send alerts.
```

File: keep/README.md [hunk 37]
```diff
@@ -0,0 +1,72 @@
...
+Need help? Contact [Keep's Support team][5].
+
+[1]: https://app.datadoghq.com/integrations/keephq
+[2]: /developers/authorization/oauth2_in_datadog/
+[3]: https://app.datadoghq.com/organization-settings/api-keys
+[4]: https://app.datadoghq.com/integrations
+[5]: mailto:rnd@keephq.dev?subject=[Datadog]%20OAuth%20Integration%20Support
+[6]: https://www.keephq.dev/
+[7]: https://app.datadoghq.com/integrations/webhooks
+[8]: https://docs.keephq.dev/providers/documentation/datadog-provider
```
Comment (alai97) [inline keep/README.md:72]:
```suggestion
[8]: https://docs.keephq.dev/providers/documentation/datadog-provider
[9]: https://app.datadoghq.com/integrations/keephq
```

File: keep/README.md [hunk 38]
```diff
@@ -0,0 +1,72 @@
...
+3. Users will be redirected back to Datadog to review and confirm the required scopes from their Datadog account
+4. Once confirmed, users will be redirected back to Keep's platform to see whether installation was successfully completed
+
+Once installed successfully, Keep will automatically create a new `Webhook` integration in your Datadog, and will modify your monitors to send alerts.
+
+## Uninstallation
+
+- Once this integration has been uninstalled, any previous authorizations are revoked. 
+- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][3].
+- Confirm that the `Webhooks` integration does not contain `keep-datadog-webhook-integration-UUID` on the [Integration page][7].
```
Comment (alai97) [inline keep/README.md:39]:
```suggestion
- Confirm that the `Webhooks` integration does not contain `keep-datadog-webhook-integration-UUID` on the [Integrations page][7].
```

File: keep/README.md [hunk 39]
```diff
@@ -0,0 +1,72 @@
...
+
+### Validation
+
+To validate Keep's integration is properly working, follow the next steps:
```
Comment (alai97) [inline keep/README.md:43]:
```suggestion
To validate that the Keep integration is properly working, follow the next steps:
```

**Review by alai97 — COMMENTED** [review]
Some more copy suggestions!

File: keep/README.md [hunk 40]
```diff
@@ -0,0 +1,74 @@
...
+
+For more information, see the [official Keep documentation][8].
+
+
+
+## Setup
+
+### Installation
+
+Use the [integration tile ][9] on your Datadog account to install the Keep integration with [OAuth2][2].
```
Comment (alai97) [inline keep/README.md:25]:
```suggestion
Use the [integration tile][9] on your Datadog account to install the Keep integration with [OAuth2][2].
```

**Review by alai97 — APPROVED** [review]
Looks good, one spacing suggestion!

**Review by emarsha94 — APPROVED** [review]

File: keep/README.md [hunk 41]
```diff
@@ -0,0 +1,74 @@
+# Agent Check: Keep
+
+## Overview
+
+[Keep][6] is an open source AIOps alert management and automation platform, which offers capabilities for consolidating, automating, and reducing noise across various parts of your observability stack. With this integration, you can harness the strength of both platforms, providing a unified and efficient approach to alert management and event correlation.
+
+Keep uses Datadog monitors data, logs, and events to correlate alerts and reduce noise.
```
Comment (steveny91) [inline keep/README.md:7]: Hmm this sentence seems a bit off to me. Maybe:

"Keep uses Datadog monitor's data..."
"Keep uses Datadog's monitoring data..."

File: keep/README.md [hunk 42]
```diff
@@ -0,0 +1,74 @@
...
+
+To validate that the Keep integration is properly working, follow the next steps:
+1. Navigate to the [Webhook Integration page][7].
+2. In the installed `Webhooks` list, look for a `Webhook` that starts with `keep-datadog-webhook-integration-UUID`.
+
+## Data Collected
+
+### Metrics
+
+Keep sends metrics about the fatigueness of specific connected monitors.
```
Comment (steveny91) [inline keep/README.md:52]: Will metrics be submitted? If so then I think you'd need a metadata.csv to properly document all the metrics.

I.e:
https://github.com/DataDog/integrations-core/blob/master/argocd/metadata.csv

Which will then render as a table in the docs and the integration tiles

File: keep/README.md (hunk 41 above)
Comment (talboren) [reply keep/README.md:7]: Well, we specifically use the monitors data (monitors definition, events, etc.) so 🤷‍♂️ I can change that, let me know what makes sense.

File: keep/README.md (hunk 42 above)
Comment (talboren) [inline keep/README.md:52]:
```suggestion
The Keep integration does not include any metrics.
```

Comment (talboren) [reply keep/README.md]: It's something we plan on adding so I removed it for now.

**Review by james-eichelbaum — APPROVED** [review]

File: keep/README.md (hunk 41 above)
Comment (steveny91) [reply keep/README.md:7]:
```suggestion
Keep uses Datadog's monitors, logs, and events data to correlate alerts and reduce noise.
```

**Review by steveny91 — APPROVED** [review]

**Pull request merged**